modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -209,6 +209,33 @@ mirror:
     targetCatalog: my-namespace/my-operator-catalog
 ----
 
+// New example; multi-arch; all versions
+[discrete]
+[id="oc-mirror-image-set-examples-multi-arch_{context}"]
+== Use case: Including all versions of OpenShift Container Platform from a minimum to the latest
+
+The following `ImageSetConfiguration` file sets the `architectures` field to `multi` to include `multi` as the platform architecture which mirrors a multi-arch release image.
+
+Previously, oc-mirror users are able to mirror x86_64 release images when no platform architecture is specified in imageSetConfig. They could also mirror other architectures individually by adding other platform architectures in the imageSetConfig.
```

Comment:
Sorry but this bit of info doesn't really add much value. Instead provide more helpful information. For example, consider answering the following ques:

- What's the default value for the `architectures` field? 
Based on the example YAML, I see that it's `amd64` 
- What are the possible values? 
`amd64`, `arm64`, `ppc64le`, `s390x` and `multi`
- What happens when you leave the `architectures` field empty? 
Previously it was `x86_64` but that's irrelevant now. So what's it today?

Alternatively, you could add callouts in the YAML file and answer the same question more specifically.

Further, for the instances of imageSetConfig either use general reference or object reference. For example, image set configuration file or `ImageSetConfiguration` object.   
If you choose to go with the backtick version for object references, make sure to also include the word "object" or "resource" or similar following the object names. 

See [API object formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references)